integration/tests/cypress_e2e.sh

```diff
@@ -13,13 +13,16 @@ do_deploy() {
     chef-automate license apply "$A2_LICENSE"
 
     case $IAM in
+        v1)
+            export CYPRESS_IAM_VERSION='v1'
```

Comment:
Adding this to each case is the same as just doing `export CYPRESS_IAM_VERSION=$IAM` outside the case, isn't it? 😉